index.html

```diff
@@ -0,0 +1,245 @@
+                            <p class="card-subtitle">CC Exhibition</p>
+                        </div>
+                        <div class="card-body">    
+                            <p class="card-text text-center">Appreciate various creations applying CC license of artists, organized from Art</p>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col-md-2 program-card">
+                    <div class="card border-0">
+                        <div class="card-body text-center">
+                            <img src="assets/main-program/program_icon_03.png" alt="eBook" class="img-fluid mb-4">
+                            <p class="card-subtitle">Forum</p>
+                        </div>
+                        <div class="card-body">    
+                            <p class="card-text text-center">Have the time to share your thoughts and opinions with experts for each topic.</p>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col-md-2 program-card">
+                    <div class="card border-0">
+                        <div class="card-body text-center">
+                            <img src="assets/main-program/program_icon_04.png" alt="eBook" class="img-fluid mb-4">
+                            <p class="card-subtitle">Workshop</p>
+                        </div>
+                        <div class="card-body">    
+                            <p class="card-text text-center">Try creating your own work using open source license rather than just watching at it.</p>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col-md-2 program-card">
+                    <div class="card border-0">
+                        <div class="card-body text-center">
+                            <img src="assets/main-program/program_icon_05.png" alt="eBook" class="img-fluid mb-4">
+                            <p class="card-subtitle">CC Ignite</p>
+                        </div>
+                        <div class="card-body">    
+                            <p class="card-text text-center">get opportunities to with CC affiliates around the world, also after the summit.</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="row pb-5 align-items-center text-center justify-content-center">
+                <div class="d-none d-md-block see-whole">
+                    SEE THE WHOLE PROGRAM
+                </div> 
+                <div class="d-block d-md-none join-summit">
+                    Join CC Global Summit 2023
+                </div> 
+            </div>
+        </div>
+    </section>
+
+    <!-- feature speakers -->
+    <section id="speakers">
+        <div class="text-center">
+            <h2 class="speakers">Featured Speakers</h2>
+        </div>
```

Comment:
- [ ] It is required to produce a close replica of the guiding template. This includes, font, typography, margin padding, among others. There's a top margin for the header `Feature Speakers` on the orignal design which is absent on your design. Kindly fix that.
- [ ] Also, there's a small underline under the header `Feature Speakers` which is also not present on your design. Kindly add that too.

### Your Design
![your-speakers](https://github.com/samuelkinyanjui/Capstone-Project_sam/assets/26597432/b0a7e7e0-1347-4151-8a87-fe7035fd3845)

### Guiding Template
![temp-speaker](https://github.com/samuelkinyanjui/Capstone-Project_sam/assets/26597432/3e9771bb-c482-4d37-bf85-2a24fc1f410a)